src/components/GrowlNotification/index.tsx

```diff
@@ -58,13 +61,11 @@ function GrowlNotification(_, ref) {
      * @param {Number} val
      */
     const fling = useCallback(
-        (val = INACTIVE_POSITION_Y) => {
+        (val = INACTIVE_POSITION_Y) =>
             Animated.spring(translateY, {
                 toValue: val,
-                duration: 80,
```

Comment:
Why was this changed? Make sure to test it throughly 😃

Reply:
This variable does not exist for Animated.spring config parameter, it didn't serve any purpose.